setup.py

```diff
@@ -12,9 +14,39 @@
     read_embree_location, \
     in_conda_env
 from distutils.version import LooseVersion
+from distutils.ccompiler import CCompiler
 import pkg_resources
 
 
+def _get_cpu_count():
+    try:
+        nthreads = os.cpu_count()
+    except AttributeError:
+        nthreads = multiprocessing.cpu_count()
```

Comment:
Out of curiosity, why can't we just use `os.cpu_count`?

Reply:
For some reason I thought `os.cpu_count()` was introduced fairly recently, but now I see it was py3.4